test_functional.py

```diff
@@ -10,7 +10,7 @@ class FunctionalTests(setuptools.Command):
     ]
 
     def run(self):
-        os.system("nose2 -s functional_tests")
+        os.system("nose2 -s functional_tests -t '.'")
```

Comment:
Seems to me this will have unexpected results if you run this script from some other directory than the directory the script is in. Should you be using `os.path.dirname(os.path.abspath(__file__))` instead?